File: 프로그래머스/0/120816. 피자 나눠 먹기 （3）/피자 나눠 먹기 （3）.py
def solution(slice, n):
    answer = n // slice  # 기본적으로 필요한 피자 판 수

    if n % slice != 0:  # n이 slice로 나누어 떨어지지 않는 경우
        answer += 1  # 추가로 한 판을 더 주문해야 모든 사람이 먹을 수 있음

    return answer  # 최종적으로 필요한 피자 판 수 반환

# 테스트 예제 실행
print(solution(7, 10))  # 2
print(solution(4, 12))  # 3

File: 프로그래머스/0/120816. 피자 나눠 먹기 （3）/피자 나눠 먹기 （3）.go
func solution(slice int, n int) int {
    // 기본적으로 필요한 피자 판 수 계산
    var answer int = n / slice
    // 나머지가 있으면 한 판 추가
    if n % slice != 0 {
        answer++
    }
    // 최종적으로 필요한 피자 판 수 반환
    return answer 
}